### ISSUE TITLE
Assert that HSReplay.net is fully translatable

### ISSUE THREAD

**Issue by beheh** [HearthSim/HSReplay.net#760]
As a non-English-speaking user, I want to be able to view HSReplay.net in my native language, so that I can fully understand the statistics and other things I'm seeing.

This story is designed as a test to demonstrate that our work on making the site localizable is complete. So we need to either try to translate all of the available strings for a particular language spoken natively / fluently by our engineering team or make use of a thorough volunteer translation (i.e., Russian) then review the site to make sure we haven't missed any strings. (If we have, we'll go back and make them extractable and then translate them and review again; if there's a good reason we can't do this within the scope of this ticket, we'll add those cases as low-priority bugs.)

Exceptions to this ticket:
- Premium modals
- Error pages
- Joust chrome
- Stripe Product Names ("HSReplay.net Premium - Monthly")
- OAuth app descriptions ("The official Hearthstone Deck Tracker")
- [Contact page](https://hsreplay.net/contact/) (which is static page)
- non-released/non-public pages (CardEditor/Admin interface)
- Currencies and Plan descriptions ($4.99/month)
- Coupons
- Testimonials

Once this ticket is complete:
- [x] All visible text on the site (with the exceptions listed above) is in the target language. This includes tooltips, filter controls, graph labels, navbar elements (including dropdowns) and the footer
- [x] All dates and numbers are formatted in a way a native speaker would expect
- [x] Any exceptions to the acceptance criteria above have been reviewed with the product owner (Andrew) and bugs have been logged for them as appropriate

**Comment by beheh** [comment]
Missing:
- table components
- sidebar
- graph labels
- Class Distribution

**Comment by beheh** [comment]
We should change this ticket:
- I don't feel we should exclude the Modals, as they are very well enumerated and can be translated like any other part of the frontend
- There's a few more pages I feel like should be excluded:
  - the admin interface
  - pages not seen by non-staff users (such as the CardEditor)
  - static pages (which mostly are the Privacy Policy and Terms of Service right now)
- In the past we've decided against formatting currencies, even though they're decimals. We should either document that here right away as an exclusion or file a separate issue for that before we head into this ticket.

**Comment by beheh** [comment]
I've thought of a not-completely-trivial missing translation: Coupon strings (when using the coupon) are assembled in dj-stripe (`coupon.human_readable`). These are strings describing the coupon, such as `100% off forever` of `$2.50 USD off once` and may be used inside strings (`You have a coupon for 100% off forever`).

**Comment by beheh** [comment]
I don't think that HearthSim/Joust#221 should be set as a blocker for this as we've specifically excluded Joust here

**Comment by beheh** [comment]
More things I am noticing:
- Stripe Product Names ("HSReplay.net Premium - Monthly")
- Human-readable OAuth scopes ("Full access to your HSReplay.net account")
- OAuth app descriptions ("The official Hearthstone Deck Tracker")
- [Contact page](https://hsreplay.net/contact/) (which is static page)
- Testimonials: We might not want these to be translated, or otherwise we should explicitely mark them as "Translated"

**Comment by jleclanche** [comment]
We can probably turn the contact page into a Help article now that we have that.

**Comment by jleclanche** [comment]
![image](https://user-images.githubusercontent.com/235410/43006062-6c5c0e8e-8c3d-11e8-8ced-26609860b4f3.png)
This will be a problem in other languages probably

**Comment by jleclanche** [comment]
![image](https://user-images.githubusercontent.com/235410/43007230-6e95e078-8c40-11e8-8645-8606978a2ae0.png)

**Comment by beheh** [comment]
That latter one is #906.

**Comment by jleclanche** [comment]
Completed
